spec/spec_with_project.rb

```diff
@@ -127,6 +127,16 @@ def refute_success_status(result)
       refute(result.status)
     end
 
+    sig { params(result: MockProject::ExecResult, snippet: String).void }
+    def assert_output_includes(result, snippet)
```

Comment:
We'd been using `stderr` and `stdout` in assertion names, for example, `assert_empty_stderr` and `assert_empty_stdout` above, so it would be much nicer if these were also `assert_stdout_includes` and `assert_stderr_includes`, as well.